beta-src/src/utils/parseSeconds.ts

```diff
@@ -0,0 +1,21 @@
+import { ParsedTime } from "../interfaces/ParsedTime";
+
+export default function parseSeconds(seconds: number): ParsedTime {
+  let timeLeft: ParsedTime = {
+    days: 0,
+    hours: 0,
+    minutes: 0,
+    seconds: 0,
+  };
+
+  if (seconds > 0) {
+    timeLeft = {
+      days: Math.floor(seconds / (60 * 60 * 24)),
```

Comment:
make 60 and 24 const outside this function. 60 would be MIN_SEC and 24 would be HOURS